lib/jets/commands/templates/skeleton/lib/core_extensions/active_record/migration_checker.rb

```diff
@@ -0,0 +1,37 @@
+module CoreExtensions
+  module ActiveRecord
+    module MigrationChecker
+      def prepare_test_db
+        require 'rails'
```

Comment:
Wondering how to do this w/o requiring rails which will define the Rails const 🤔. Related: https://community.rubyonjets.com/t/why-does-jets-define-rails-module/29

Also, wondering how to do it w/o generating a file as part of the skeleton template 🧐

Reply:
@tongueroo thanks for your review

> Wondering how to do this w/o requiring rails which will define the Rails const

Should be fixed in this ffe6e61f1584033429da5b87b551998c2233d608

> how to do it w/o generating a file as part of the skeleton template

where would you recommend to put the migration checker? I have tried to put it inside the `lib/jets/spec_helpers.rb` (c09c2abd3365d9e3f2cc643c278c208eb800ecdb) but it fails the [build](https://circleci.com/gh/tongueroo/jets/2678)

Reply:
> RE: where would you recommend to put the migration checker?

Unsure. Spec helpers seems like a fine place. Maybe hack it and don’t run it as part of the spec run?

> RE: Not generating the file as part of the skeleton

Thanks for changing that. It’s better to have more control of the overrides in case there are changes in the future. 

Would also move it from lib/core_ext to overrides/rails. Have the overrides/monkey patching close to each other.

Reply:
I have move it into `lib/jets/overrides/active_record` because most of the code is coming from active record module. Also i have added a new ENV config `SKIP_MIGRATION_CHECK` to control whether we should run the migration checking or not.
Please help to take a look again. :smile: